src/dispatch/group/flows.py

```diff
@@ -52,41 +52,45 @@ def create_group(
         weblink=external_group["weblink"],
     )
     group = create(db_session=db_session, group_in=group_in)
-    obj.groups.append(group)
+    subject.groups.append(group)
 
     if group_type == GroupType.tactical:
-        obj.tactical_group_id = group.id
+        subject.tactical_group_id = group.id
     else:
-        obj.notifications_group_id = group.id
+        subject.notifications_group_id = group.id
 
-    db_session.add(obj)
+    db_session.add(subject)
     db_session.commit()
 
-    obj_type = get_table_name_by_class_instance(obj)
-    if obj_type == "case":
+    subject_type = get_table_name_by_class_instance(subject)
```

Comment:
I think event logging should take place in the calling function e.g. `create_case_flow` and `create_incident_flow`

Reply:
I moved the logging out to the calling function. I could see value in having logs in the method that was called (or both places), this case `update_group`, so people get some logging for free across the board. I didn't see incident flows using the group flow here, so I just removed it as well. 

cc @mvilanova in case you have any thoughts, since I think you added these log statements.